### ISSUE TITLE
Finish wireframe

### ISSUE THREAD

**Issue by ejc11** [knights-who-say-c/scrum-hub#28]
**Task Tests**

*Test 1*
1. Open Project on Figma
Link: https://www.figma.com/file/EpxvbstUiQzbhE6mFSYRbx/Untitled?node-id=0%3A1
2. Click button the play button in the top right corner
3. Click the first name spot in the login screen
4. Click Submit
5. After Submit is pressed, the main screen with "Create new project" should be present

*Test 2*
1. Open Project on Figma
Link: https://www.figma.com/file/EpxvbstUiQzbhE6mFSYRbx/Untitled?node-id=0%3A1
2. Click button the play button in the top right corner
3. Click the "Log In" button on the top right side of the screen
3. Click the email spot in the login screen
4. Click Submit
5. After Submit is pressed, the main screen with "Create new project" should be present

*Test 3*
1. Open Project on Figma
Link: https://www.figma.com/file/EpxvbstUiQzbhE6mFSYRbx/Untitled?node-id=0%3A1
2. Click button the play button in the top right corner
3. Click the "Log In" button on the top right side of the screen
3. Click the email spot in the login screen
4. Click Submit
5. Click "Due dates" on the left column
6. After "Due dates" is pressed, the due date should be present
7. Click "Home" at the top of the page to go to the project home screen
8. After "Home" is pressed, the main screen with "Create new project" should be present

*Test 4*
1. Open Project on Figma
Link: https://www.figma.com/file/EpxvbstUiQzbhE6mFSYRbx/Untitled?node-id=0%3A1
2. Click button the play button in the top right corner
3. Click the "Log In" button on the top right side of the screen
3. Click the email spot in the login screen
4. Click Submit
5. Click "Create new project" on the main page
6. Click "Essay" or "Book" or "Code" or "Manage Offline Project"
7. After a type of project is pressed, the ScrumHub board should be present

*Test 5*
1. Open Project on Figma
Link: https://www.figma.com/file/EpxvbstUiQzbhE6mFSYRbx/Untitled?node-id=0%3A1
2. Click button the play button in the top right corner
3. Click the "Log In" button on the top right side of the screen
3. Click the email spot in the login screen
4. Click Submit
5. Click "Create new project" on the main page
6. Click "Essay" or "Book" or "Code" or "Manage Offline Project"
7. After a type of project is pressed, the ScrumHub board should be present
8. Click "Add New Issue" on the top right side of the board
9. After the button is pressed, the new issue tab should be present
10.  Click on the box under the "Tittle" 
11. After a box is pressed, the tittle and description should be present
12. Click "Labels" 
13. Click "User Story" 
14. Click "Assignee" 
15. Click "Joe"
16. Click "Due Date"
17. Click "Submit"
18. After submit is pressed, the ScrumHub board should be present with the new User Story on it.

*Test 6*
1. After Task 5, the User Story should present on the board
2. Click the on the task box, to move it over to the next column
3. After the task box is pressed, the box should be moved into the Planned Column
4. Click the on the task box, to move it over to the next column
5. After the task box is pressed, the box should be moved into the In Progress Column
6. Click the on the task box, to move it over to the next column
7. After the task box is pressed, the box should be moved into the Testing Column
8. Click the on the task box, to move it over to the next column
9. After the task box is pressed, the box should be moved into the Completed Column
10. Click the on the task box, to move it over to the next column
11. After the task box is pressed, the box should be moved into the Closed Column

*Test 7*
1. Open Project on Figma
Link: https://www.figma.com/file/EpxvbstUiQzbhE6mFSYRbx/Untitled?node-id=0%3A1
2. Click button the play button in the top right corner
3. Click the "Log In" button on the top right side of the screen
3. Click the email spot in the login screen
4. Click Submit
5. Click "Due dates" on the left column
6. Click "Tasks" on the left column
7. After the "Tasks" is pressed, the Tasks screen should be present with all the users tasks

*Test 8*
1. Open Project on Figma
Link: https://www.figma.com/file/EpxvbstUiQzbhE6mFSYRbx/Untitled?node-id=0%3A1
2. Click button the play button in the top right corner
3. Click the "Log In" button on the top right side of the screen
3. Click the email spot in the login screen
4. Click Submit
5. Click "Due dates" on the left column
6. Click "Meetings Tool" on the left column
7. After the "Meetings Tool" is pressed, the Meetings Tool should be present with a list of the collaborators
8. Click on the box in the Joe and Monday section to put a check mark there to represent the user is available that day for a meeting
9. After it is pressed, the Meetings Tool should be present with a list of the collaborators and check marks for availabity should be present

*Test 9*
1. Open Project on Figma
Link: https://www.figma.com/file/EpxvbstUiQzbhE6mFSYRbx/Untitled?node-id=0%3A1
2. Click button the play button in the top right corner
3. Click the "Log In" button on the top right side of the screen
3. Click the email spot in the login screen
4. Click Submit
5. Click "Due dates" on the left column
6. Click "Settings" on the left column
7. After the "Settings" is pressed, the settings screen should be present
8. Click on "Log Out" button on the bottom right of the screen
9. After it is pressed, the log in screen should be present

*Test 10*
1. Open Project on Figma
Link: https://www.figma.com/file/EpxvbstUiQzbhE6mFSYRbx/Untitled?node-id=0%3A1
2. Click button the play button in the top right corner
3. Click the "Log In" button on the top right side of the screen
3. Click the email spot in the login screen
4. Click Submit
5. Click "Add Collaborator" on the top right
6. After the "Add Collaborator" is pressed, the Add Collaborator screen should be present
7. Click the box under "Enter Collaborator"
8. Click the "Add" button
9. After the "Add" button is pressed, the screen should tell you that that collaborator should be add
10. Click the "Next" button
11. After the "Next" button is pressed, the "Create New Project" screen should be present
